compiler/rustc_mir/src/transform/check_consts/mod.rs

```diff
@@ -77,6 +77,8 @@ pub fn is_lang_panic_fn(tcx: TyCtxt<'tcx>, def_id: DefId) -> bool {
     Some(def_id) == tcx.lang_items().panic_fn()
```

Comment:
```suggestion
    // We can allow calls to these functions because `hook_panic_fn` in
    // `const_eval/machine.rs`ensures the calls are handled specially.
    // Keep in sync with what that function handles!
    Some(def_id) == tcx.lang_items().panic_fn()
```
I know this is pre-existing but I think a bit more documentation couldn't hurt here.